comeet/Data/Authenticator/AuthenticatorADALImplementer.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  AuthenticatorADALImplementer.swift
+//  comeet
+//
+//  Created by Ricardo Contreras on 3/6/17.
+//  Copyright © 2017 teamawesome. All rights reserved.
+//
+
+import Foundation
+import ADAL
+
+class AuthenticatorADALImplementer : AuthenticatorProtocol
```

Comment:
@kb000 Yes. I'll write a test for `AuthenticatorADALImplementer` and `LoginViewModel`